provider/ansible.rb

```diff
@@ -75,8 +75,9 @@ def initialize(config, api)
       def python_type(prop)
         prop = Module.const_get(prop).new('') unless prop.is_a?(Api::Type)
         # All ResourceRefs are dicts with properties.
+        # We're assuming that all ResourceRefs in the list act the same.
```

Comment:
what does 'act the same' mean in this context?

Reply:
All virtual or all not virtual. That was unclear. Made a better comment.